frontend/src/components/Voting/style.css

```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap");
+
+body,
```

Comment:
css could break mui component styling especially if you override body, h2 etc

Better if we can use mui built in way of tweaking css

Reply:
This shall be outdate